source/new-ia/by-role/system-admin/clustering/cluster-troubleshooting.rst

```diff
@@ -0,0 +1,120 @@
+        NULL, '172.31.70.118')'
+	    at org.craftercms.studio.impl.v2.dal.cluster.DbPrimaryReplicaClusterSynchronizationServiceImpl.checkForErrors(DbPrimaryReplicaClusterSynchronizationServiceImpl.java:598) ~[classes/:4.0.2-SNAPSHOT]
+	    at org.craftercms.studio.impl.v2.dal.cluster.DbPrimaryReplicaClusterSynchronizationServiceImpl.waitForLocalReplicaToSync(DbPrimaryReplicaClusterSynchronizationServiceImpl.java:571) ~[classes/:4.0.2-SNAPSHOT]
+	    at org.craftercms.studio.impl.v2.dal.cluster.DbPrimaryReplicaClusterSynchronizationServiceImpl.synchronizeStartup(DbPrimaryReplicaClusterSynchronizationServiceImpl.java:270) ~[classes/:4.0.2-SNAPSHOT]
+	    at org.craftercms.studio.impl.v2.dal.cluster.DbPrimaryReplicaClusterAwareMariaDB4jSpringService.start(DbPrimaryReplicaClusterAwareMariaDB4jSpringService.java:51) ~[classes/:4.0.2-SNAPSHOT]
+	    at ch.vorburger.mariadb4j.MariaDB4jService.postConstruct(MariaDB4jService.java:64) ~[mariaDB4j-core-2.5.3.jar:?]
+	    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
+        ...
+
+An email will also be sent to the configured list of recipients to inform them of the failure.
+
+See the :ref:`newIa-setup-a-two-node-cluster-with-studio` article then scroll to the
+:ref:`failure notification properties <newIa-cluster-startup-failure-notification-config>` section
+for more information on how to configure the list of recipients to be informed in case of a
+startup failure in the cluster.
+
+This section discusses how to fix the sync failure in your cluster.
+
+
+.. raw:: html
+
+   <hr>
+
+-----------------------
+Fixing the Sync Failure
+-----------------------
+
+The first thing to do when a sync failure happens is to figure out whether the sync failure is in the DB or Git.
+The email sent to configured recipients when the sync failure happened will indicate whether it's a DB or a Git
+sync failure.  From the logs, you can also determine if it was a DB or a Git sync failure.
+
+^^^^^^^^^^^^^^^
+DB sync failure
+^^^^^^^^^^^^^^^
+
+For a DB sync failure, the logs will contain a message like below:
+
+.. code-block:: text
+
+   ...
+   Failed to start DB replica:
+   ...
+
+as seen :ref:`above <newIa-cluster-troubleshooting-git-sync-fail-log>` and the following email will be sent if configured:
+
+.. image:: /_static/images/system-admin/cluster-db-sync-fail.webp
+   :alt: CrafterCMS - Studio Enterprise Clustering DB sync failure email
+   :width: 35%
+
+|
+
+After reviewing the node states (logs and the database), the fix can be any valid intervention on the database.
+After fixing the database, the node may then be restarted.
+
+If an admin reviews the node states and thinks everything is fine but still receives DB sync errors, the admin
+may decide if MariaDB should ignore those errors and continue. To ignore the errors, a manual intervention is
+required and may be done by following the instructions `here <https://mariadb.com/kb/en/set-global-sql_slave_skip_counter/>`__
```

Comment:
If there is a startup failure, Studio will stop and MariaDB will be down. Can we add here instructions to start it up in order to perform the manual fix?